docs/src/main/resources/markdown/developers/architecture/osgi/configadmin.md

```diff
@@ -1,12 +1,21 @@
----
-title: Configuration Admin
----
 
-# Overview
+# Configuration Admin
+
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="../../index.php">Developers Guide</a></li>
```

Comment:
I see this new navigation block turning up. While I agree we need navigation, I don't think this is a good approach to manually add it to every file - it will be a bit of a nightmare to maintain this. As I mentioned in the forum post, I would ideally like to see something that auto builds based on the hierarchy of the files rather than manually adding this to every file.

I don't know how to manage that, but I think we should work that out at this stage rather than edit all the files and  then have to change them all.